internal/bminventory/inventory_v2_handlers.go

```diff
@@ -340,10 +340,8 @@ func (b *bareMetalInventory) v2uploadLogs(ctx context.Context, params installer.
 		if err != nil {
 			return err
 		}
-
-		b.eventsHandler.AddEvent(ctx, dbHost.InfraEnvID, params.HostID, models.EventSeverityInfo,
-			fmt.Sprintf("Uploaded logs for host %s cluster %s",
-				hostutil.GetHostnameForMsg(&dbHost.Host), params.ClusterID.String()), time.Now())
+		eventgen.SendHostLogsUploadedEvent(ctx, b.eventsHandler, &params.ClusterID, *params.HostID, params.ClusterID,
+			hostutil.GetHostnameForMsg(&dbHost.Host))
```

Comment:
shouldn't `dbHost.InfraEnvID` being provided instead of the second `params.ClusterID` ?